src/legacy/core_plugins/kibana/public/dashboard/panel/panel_utils.ts

```diff
@@ -82,38 +109,40 @@ export class PanelUtils {
     return panel;
   }
 
-  static parseVersion(version = '6.0.0') {
+  public static parseVersion(version = '6.0.0') {
     const versionSplit = version.split('.');
     if (versionSplit.length < 3) {
-      throw new Error(i18n.translate('kbn.dashboard.panel.invalidVersionErrorMessage', {
-        defaultMessage: 'Invalid version, {version}, expected {semver}',
-        values: {
-          version,
-          semver: '<major>.<minor>.<patch>',
-        },
-      }));
+      throw new Error(
+        i18n.translate('kbn.dashboard.panel.invalidVersionErrorMessage', {
+          defaultMessage: 'Invalid version, {version}, expected {semver}',
+          values: {
+            version,
+            semver: '<major>.<minor>.<patch>',
+          },
+        })
+      );
     }
     return {
       major: parseInt(versionSplit[0], 10),
-      minor: parseInt(versionSplit[1], 10)
+      minor: parseInt(versionSplit[1], 10),
     };
   }
 
-  static initPanelIndexes(panels) {
+  public static initPanelIndexes(panels: PanelState[]) {
```

Comment:
could add type annotation for return value as well?

Reply:
Adding void

Reply:
fc7f3db